### ISSUE TITLE
ImageDeserializer cannot locate Image

### ISSUE THREAD

**Issue by FlyinTeller** [Microsoft/CNTK#2551]
I am currently trying to classify some images using CNTK. However I am having trouble with even the first steps. I just cannot get the ImageDeserializer to properly open my Images. I even tried running 

`CNTK_201A_CIFAR-10_DataLoader.ipynb`
followed by 
 `CNTK_201B_CIFAR-10_ImageHandsOn.ipynb`

with the same result. I am getting something like this:

> ---------------------------------------------------------------------------
> RuntimeError                              Traceback (most recent call last)
> <ipython-input-13-3970f8295ba5> in <module>()
>       2                           reader_test,
>       3                           max_epochs=5,
> ----> 4                           model_func=create_basic_model)
> 
> <ipython-input-12-c9e0b256eb55> in train_and_evaluate(reader_train, reader_test, max_epochs, model_func)
>      55         while sample_count < epoch_size:  # loop over minibatches in the epoch
>      56             data = reader_train.next_minibatch(min(minibatch_size, epoch_size - sample_count), 
> ---> 57                                                input_map=input_map) # fetch minibatch.
>      58             trainer.train_minibatch(data)                                   # update model with it
>      59 
> 
> ~\AppData\Local\Continuum\miniconda3\envs\py36cntk\lib\site-packages\cntk\internal\swig_helper.py in wrapper(*args, **kwds)
>      67     @wraps(f)
>      68     def wrapper(*args, **kwds):
> ---> 69         result = f(*args, **kwds)
>      70         map_if_possible(result)
>      71         return result
> 
> ~\AppData\Local\Continuum\miniconda3\envs\py36cntk\lib\site-packages\cntk\io\__init__.py in next_minibatch(self, minibatch_size_in_samples, input_map, device, num_data_partitions, partition_index)
>     327                                             minibatch_size_in_samples,
>     328                                             num_data_partitions,
> --> 329                                             partition_index, device)
>     330 
>     331         if not mb:
> 
> ~\AppData\Local\Continuum\miniconda3\envs\py36cntk\lib\site-packages\cntk\cntk_py.py in get_next_minibatch(self, *args)
>    2924 
>    2925     def get_next_minibatch(self, *args):
> -> 2926         return _cntk_py.MinibatchSource_get_next_minibatch(self, *args)
>    2927 MinibatchSource_swigregister = _cntk_py.MinibatchSource_swigregister
>    2928 MinibatchSource_swigregister(MinibatchSource)
> 
> RuntimeError: Cannot open file 'C:\******\CNTK\Tutorials\data\CIFAR-10\train\21258.png'
> 
> [CALL STACK]
> Failed to generate CALL STACK. SymInitialize() failed with error The parameter is incorrect.

The image file is there and accessible, I can even open it from any other python program. I am also able to run other examples from the tutorial just fine, so I am am quite stuck what the issue might be here

I am using a miniconda Python 3.6 VirtualEnv on a Windows 10 machine if that is of interest

**Comment by eldakms** [comment]
What is ******  in the path? Are you sure your path is correct and the mapping file is ASCII?
Could you publish the file here? Thanks!

 Cannot open file 'C:******\CNTK\Tutorials\data\CIFAR-10\train\21258.png'

**Comment by FlyinTeller** [comment]
I omitted the part of the path containing my username etc. The apth is correct. I checked by copy pasting it into a 

`open("...")`
command in a python interpreter
Attached is the file you asked for. 
![21258](https://user-images.githubusercontent.com/7376593/32060565-5eeebc74-ba6f-11e7-8b4a-566137ff14a6.png)

**Comment by FlyinTeller** [comment]
Just realized you probably meant the map file:

[test_map.txt](https://github.com/Microsoft/CNTK/files/1421799/test_map.txt)
[train_map.txt](https://github.com/Microsoft/CNTK/files/1421801/train_map.txt)

**Comment by cha-zhang** [comment]
If you can run other examples just fine, then it's still the path issue. If you are doing CIFAR-10, why don't you run the script here: 
https://github.com/Microsoft/CNTK/tree/master/Examples/Image/DataSets/CIFAR-10

**Comment by FlyinTeller** [comment]
Thanks for your comment cha-zhang. I am actually not doing Cifar-10, just tried to use the tutorial as a starting point to see what might be wrong with my code and then realized that it didn't work either. I ran the data loader 201A and then 201B right after, so the paths are just what was automatically created. I have switched now to creating my own minibatch source as that should be more convenient for me in the longrun anyway. Closing this for now.
